language-html.md

```diff
@@ -16,20 +22,78 @@ style–guides about HTML.
 >
 ```
 
+#### Self-closing tags (void elements):
+
+In the HTML 5 specification the `/` character is valid but has no effect for void elements such as `<br />, <hr />, <img />, <input />`.
+So in our Rabbit team we decided not to close such void elements , i.e.
+```html
+<img class="my-class" id="void-element"
+    src="please-do-not-use-slash-to-close-this-element"
+>
```

Comment:
@PikaboyZ 
Yeap, I saw it.
This is not new rule actually - we had it before. And I vote for it - it is much easier to read code.